### ISSUE TITLE
Feature Request - container leaf nodes-->CSV

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#4692]
ref: https://github.com/ArctosDB/arctos/issues/4681

Find Container / See all collection objects in this container needs a CSV option

**Comment by Jegelewicz** [comment]
Set priority based on associated issue.

**Comment by campmlc** [comment]
Thanks, yes agree priority high.

**Comment by dustymc** [comment]
Is this still needed, and is the request for what's currently visible in "See all collection objects in this container"?

**Comment by campmlc** [comment]
Yes, please - the request is for what is currently visible in "see all collection objects in this container", thanks!

**Comment by mkoo** [comment]
This is stale and will be closed
